.cloud-build/execute_changed_notebooks_cli.py

```diff
@@ -47,6 +52,12 @@ def str2bool(v):
     required=False,
     default=100,
 )
+parser.add_argument(
```

Comment:
To make it explicit that this needs to be unique to a build, replace this with `build_id` and make it required.

Reply:
done